static/partials/language-specific-deploy/go.md

```diff
@@ -1,137 +1,98 @@
+
+### Modern Go project structure
+There are multiple ways to build/run a Go application, and this has evolved over the years. In its modern form a Go project can be a:
+- `Package`: one or more `.go` files you can `build` or `run`. `main` package and `main()` function are the default entry point
+- `Module`: one or more packages you can `install`, defined in a `go.mod` file (`go.sum` for checksums)
+- `Workspace`: one or more modules seamlessly combined, defined in a `go.work` file
+
+A module can be installed locally or from a remote repository if you pass its URL to the `install` command. A `Makefile` is sometimes used to define how a Go project is built, run and/or cleaned. The lightest form of a Go project is a `main.go` file to build. Some years ago, the `src/` folder was often used for source code, but using the `cmd/` folder instead is now a common practice. 
+
+If you want to limit from where a package can be imported it [should placed](https://docs.google.com/document/d/1e8kOo3r51b2BWtTs_1uADIA5djfXhPT36s6eHVRIvaU/edit) in a folder named `ìnternal/`. Access to functions in `.go` files is defined depending [on their name](https://go.dev/tour/basics/3): if it starts with a capital letter it's a public functions, if not it's a private function.
+
+For a complete project, a common files/folders organization can be:
+{{< filetree/container >}}
+  {{< filetree/folder name="application-root/" >}}
+    {{< filetree/file name="go.work" >}}
+    {{< filetree/file name="Makefile" >}}
+    {{< filetree/folder name="cmd/" >}}
+      {{< filetree/file name="main.go" >}}
+      {{< filetree/file name="other-file.go" >}}
+      {{< filetree/file name="other-package.go" >}}
+      {{< filetree/file name="…" >}}
+    {{< /filetree/folder >}}
+        {{< filetree/folder name="module/" >}}
+          {{< filetree/file name="go.mod" >}}
+          {{< filetree/file name="go.sum" >}}
+          {{< filetree/file name="main.go" >}}
+          {{< filetree/file name="other-module-file.go" >}}
+            {{< filetree/folder name="internal/" >}}
+              {{< filetree/folder name="internal-package/" >}}
+                {{< filetree/file name="internal-package-file.go" >}}
+              {{< /filetree/folder >}}
+            {{< /filetree/folder >}}
+      {{< /filetree/folder >}}
+  {{< /filetree/folder >}}
+{{< /filetree/container >}}
+
+### Go build and deploy on Clever Cloud
+In such a situation, our strategy is to let the user choose how to build/run its application and make the deployment easy anyway. At first, we used the `goget` method, which is now deprecated. Thus, you can now use `gobuild` (for packages), `gomod` (for modules) or a configuration file. The latter will allow you to define a `Makefile` for custom build and a `main` executable to start the application.
+
+{{< callout type="info" >}}
+  If the required Go version declared in the `go.mod` is superior to the version built in the instance, it will be automatically updated.
```

Comment:
Withe the limit of the exherbo packaged version ?

Reply:
Nope